react-app/src/store/items.js

```diff
@@ -91,20 +107,23 @@ const itemSlice = createSlice({
     initialState,
     extraReducers: (builder) => {
       builder.addCase(createItem.fulfilled, (state, action) => {
-        state.items.entities.items[action.payload.id] = action.payload;
+        state.entities.items[action.payload.id] = action.payload;
       });
       builder.addCase(getItems.fulfilled, (state, action) => {
         const items = {}
         action.payload.forEach((item) => {
             items[item.id] = item
         })
-        state.items.entities.items = items;
+        state.entities.items = items;
       });
+      builder.addCase(getAnItem.fulfilled, (state, action) => {
+        state.entities.item = action.payload
```

Comment:
The item should be obtained from the `state.entities.items` object.

If you change this to `state.entities.items[action.payload.id] = action.payload;` you can then use a selector to get the item with that id from the `items` object.